### PR TITLE
Aion task

### PR DESCRIPTION
**Pull request by powimod**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: org.gnome.powimod.aion-task.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+runtime-version: '44'
+sdk: org.gnome.Sdk
+
+command: aion-task
+
+finish-args:
+  - '--share=ipc'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+  - '--filesystem=xdg-data/aion-task/:create'
```
Comment (hfiguiere) [inline org.gnome.powimod.aion-task.yml:12]: it shouldn't need this at all.
arbitrary `xdg-data` is in general not allowed.

File: org.gnome.powimod.aion-task.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+app-id: org.gnome.powimod.aion-task
```
Comment (hfiguiere) [inline org.gnome.powimod.aion-task.yml:1]: the app-id should be in the `org.gnome.gitlab` namespace.

**Comment by hfiguiere** [comment]
bot, build org.gnome.powimod.aion-task

File: org.gnome.powimod.aion-task.yml (hunk 1 above)
Comment (hfiguiere) [reply org.gnome.powimod.aion-task.yml:12]: This is the build failure:
```
        "finish-args-unnecessary-xdg-data-access"
```

But again I don't think you need it. If you do it's a bug, gtk is supposed to give you the right directories.

**Comment by powimod** [comment]
New commit with release 0.8.6 should solve the two issues

**Comment by hfiguiere** [comment]
the manifest file name must match the id.

**Comment by hfiguiere** [comment]
it no longer does.

**Comment by powimod** [comment]
Sorry...
Manifest file was renamed to match App ID.

**Comment by hfiguiere** [comment]
bot, build org.gnome.gitlab.powimod.aion-task

**Comment by hfiguiere** [comment]
seems the appstream file isn't installed.

**Comment by hfiguiere** [comment]
I can't find one upstream either.

**Comment by powimod** [comment]
I added metainfo file in the release 0.8.7

**Comment by hfiguiere** [comment]
bot, build org.gnome.gitlab.powimod.aion-task

**Comment by hfiguiere** [comment]
```
org.gnome.gitlab.powimod.aion-task: AppData problem: tag-invalid : <release> versions are not in order [0.8.7 before 0.9.0]
```

**Comment by powimod** [comment]
in 0.9.1 release, meta-info file was updated to reorder release versions

**Comment by barthalion** [comment]
bot, build org.gnome.gitlab.powimod.aion-task

**Comment by hfiguiere** [comment]
```
• style-invalid         : Not enough <screenshot> tags, minimum is 1
```

**Comment by powimod** [comment]
Screenshots are added in release 0.9.2

**Comment by barthalion** [comment]
bot, build org.gnome.gitlab.powimod.aion-task

**Pull request closed without merging**